CMakeLists.txt

```diff
@@ -354,7 +354,14 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 if (UNIX)
     target_compile_options(${PROJECT_NAME} PRIVATE -Wall -Wno-sign-compare -Wno-unused-function -Werror ${LLVM_CPP_FLAGS})
     # Security options
-    target_compile_options(${PROJECT_NAME} PRIVATE -fstack-protector-strong -fno-strict-float-cast-overflow
+    if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+        # Using the clang compiler
+        set(COMPILER_SPECIFIC_FLAGS -fno-strict-float-cast-overflow)
+    else()
+        # Assume that we are using GCC
+        set(COMPILER_SPECIFIC_FLAGS -fno-sanitize=float-cast-overflow)
```

Comment:
I think the right option for gcc is "-fno-strict-overflow"

Reply:
Hmm, I've tried googling both flags and it seems like both Clang and GCC supports `-fno-strict-overflow` and `-fno-sanitize=float-cast-overflow`.

I didn't manage to find a elaborate explanation for `no-strict-float-cast-overflow` and the difference with `no-sanitize=float-cast-overflow`.

Perhaps this is a case were we don't really need this `if/else` at all and can just use flags that are supported by both (with the same end result)?

Reply:
Right. Let's keep -fno-strict-overflow for both Clang and GCC.
Thanks for your findings!

Reply:
Done! :)

Reply:
Since our CI failed after adding this option I've dived into it and found out that we do not need to use -fno-strict-overflow if -fwrapv is used. So I removed -fno-strict-overflow.